powerline/listers/i3wm.py

```diff
@@ -0,0 +1,78 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+
+
+conn = None
+
+
+@requires_segment_info
```

Comment:
I do not see `segment_info` dictionary used anywhere here. But it should be used: specifically you should create a copy of `segment_info` and put output there like other listers do, *not* return a dictionary with just output.